tests/cargo-kani/kani-tokio/src/io_write.rs

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Modifications Copyright Kani Contributors
+// See GitHub history for details.
+
+// Original copyright tokio contributors.
+// origin: tokio/tests/tokio/
+
+
+#![warn(rust_2018_idioms)]
+#![cfg(feature = "full")]
+
+use tokio::io::{AsyncWrite, AsyncWriteExt};
+use tokio_test::assert_ok;
+
+use bytes::BytesMut;
+use std::io;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+#[kani::proof] fn test() {
+    assert!(false);
```

Comment:
I'm assuming this was to debug the issue of Kani ignoring this file due to the `#![cfg()]`

Reply:
Yes, indeed. This PR is not ready for review yet.